src/coreclr/utilcode/ilformatter.cpp

```diff
@@ -478,7 +478,7 @@ const BYTE* ILFormatter::formatStatement(const BYTE* instrPtr, OutString* out) {
             DO_BR: {
                 size_t target = (instrPtr - start) + inlineArg.i;
                 setTarget(target, stackDepth());
-                result << "goto IL_"; result.hex(static_cast<unsigned __int64>(target), 4, OutString::zeroFill);
+                result << "goto IL_"; result.hex(static_cast<uint64_t>(target), 4, OutString::zeroFill);
```

Comment:
Whether we use `static_cast`/`reinterpret_cast` or `C-style` casts is fairly inconsistent

Do we have one we want to normalize on moving forward, afair `static_cast` is preferred where possible for the additional language correctness validation